blue-xdma/script/sim.tcl

```diff
@@ -0,0 +1,7 @@
+open_project ./build/xdma_ex/xdma_ex.xpr
```

Comment:
no hard-coded path, use env var instead

Reply:
Currently, I am still exploring how to use variables instead of hard-coded paths. However, obtaining this path from external sources is proving to be challenging. I will make the necessary modifications in the future.